database/src/main/scala/cromwell/database/migration/metadata/table/ExecutionTableMigration.scala

```diff
@@ -0,0 +1,269 @@
+package cromwell.database.migration.metadata.table
+
+import cromwell.database.migration.metadata.MetadataCustomSql
+import MetadataCustomSql._
+
+/**
+  * Transform data from the EXECUTION table to metadata
+  */
+class ExecutionTableMigration extends MetadataCustomSql {
+
+  override def queries: Array[String] = {
+    Array(
+      s"""INSERT INTO METADATA_JOURNAL (
+        WORKFLOW_EXECUTION_UUID,
+        METADATA_KEY,
+        CALL_FQN,
+        JOB_SCATTER_INDEX,
+        JOB_RETRY_ATTEMPT,
+        METADATA_VALUE,
+        METADATA_VALUE_TYPE,
+        METADATA_TIMESTAMP
+      )
+        SELECT
+        WORKFLOW_EXECUTION_UUID,
+      'start',
+    CALL_FQN,
+    IDX,
+    ATTEMPT,
+    DATE_FORMAT(e.START_DT, '%Y-%m-%dT%T.%f$Offset'),
```

Comment:
I double checked in the code, but from what I see we inject
`new Timestamp(new Date().getTime)` ([here](https://github.com/broadinstitute/cromwell/blob/0.19_hotfix/engine/src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala#L740) + [here](https://github.com/broadinstitute/cromwell/blob/0.19_hotfix/engine/src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala#L46)) as start and end times, which on my machine returns local time:

```scala
scala> import java.sql.Timestamp
import java.sql.Timestamp
scala> import java.util.Date
import java.util.Date
scala> new Timestamp(new Date().getTime)
res0: java.sql.Timestamp = 2016-09-02 18:08:27.173

```